src/microbe_stage/Microbe.Interior.cs

```diff
@@ -1548,7 +1548,7 @@ private void HandleDigestion(float delta)
     private void HandleSlimeSecretion(float delta)
     {
         // Ignore if we have no slime jets
-        if (SlimeJets.Count < 0)
+        if (SlimeJets.Count == 0)
```

Comment:
```suggestion
        if (SlimeJets.Count < 1)
```
I'd write this in defensive programming style like this that I've used elsewhere in the game as well.